src/jsCodegen/compileXPathToJavaScript.ts

```diff
@@ -42,13 +43,14 @@ function compileXPathToJavaScript(
 	let ast;
 	if (typeof selector === 'string') {
 		const expressionString = normalizeEndOfLines(selector);
-		const parserOptions = {
+		const parserOptions: CompilationOptions = {
 			allowXQuery:
 				options['language'] === Language.XQUERY_3_1_LANGUAGE ||
 				options['language'] === Language.XQUERY_UPDATE_3_1_LANGUAGE,
 			// Debugging inserts xs:stackTrace in the AST, but this is not supported
 			// yet by the js-codegen backend.
 			debug: false,
+			version: 3.1,
```

Comment:
Should this depend on options?

Reply:
JS codegen does not do any fancy XPath 4 things, so I'm kind of inclined to make this simple: just assume 3.1 when compiling to JS

Reply:
but the `options['language']` is right there 😛

Reply:
aiaiaiai. right as always. fixed it